public/app/percona/shared/components/Elements/FeatureLoader/FeatureLoader.tsx

```diff
@@ -28,6 +29,7 @@ export const FeatureLoader: FC<FeatureLoaderProps> = ({
     try {
       const settings = await SettingsService.getSettings(generateToken(GET_SETTINGS_CANCEL_TOKEN), true);
       setFeatureEnabled(!!settings[featureFlag]);
+      onSettingsLoaded?.(settings);
```

Comment:
what's the meaning of the dot here?

Reply:
`?.` it's the optional chaining operator. We normally use it with object properties like `obj.person?.name` but it also works with functions. So instead of check if the function is defined with something like `onSettingsLoaded && onSettingsLoaded()` we can just use optional chaining `onSettingsLoaded?.()`. If it's not defined it won't call the function.